src/appleseed/foundation/utility/stampedptr.h

```diff
@@ -0,0 +1,146 @@
+// This code is taken from Facebook's Folly library,
```

Comment:
When we import a foreign source file into our source tree, we should keep it as unmodified as possible in order to (1) make updating it from upstream easier, (2) avoid introducing bugs.

Reply:
> When we import a foreign source file into our source tree, we should keep it as unmodified

The class is small and code will not likely change. We also have a test to check all is ok.
It was nicer to change it to use our own coding conventions.